libs/SalesforceSDKCore/SalesforceSDKCore/Classes/SmartStore/SFSmartStoreDatabaseManager.h

```diff
@@ -46,6 +46,8 @@ extern NSString * const kSFSmartStoreDbErrorDomain;
  */
 + (SFSmartStoreDatabaseManager *)sharedManagerForUser:(SFUserAccount *)user;
 
++ (SFSmartStoreDatabaseManager *)sharedGlobalManager;
```

Comment:
Yes.  The database manager (for users too) doesn't work with a specific database—it works with all databases for a given context.